engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -956,12 +956,13 @@ void paladin_t::generate_action_prio_list_prot()
 
   std -> add_action( this, "Shield of the Righteous" , "if=debuff.judgment.up" );
   std -> add_action( this, "Shield of the Righteous" , "if=holy_power=5|buff.holy_avenger.up|holy_power=4&talent.sanctified_wrath.enabled&buff.avenging_wrath.up" );
+  std -> add_action( this, "Hammer of Wrath", "if=runeforge.the_mad_paragon|runeforge.vanguards_momentum|covenant.venthyr&cooldown.ashen_hallow.remains>210" );
   std -> add_action( this, "Judgment", "target_if=min:debuff.judgment.remains,if=charges=2|!talent.crusaders_judgment.enabled" );
   std -> add_action( this, "Hammer of Wrath" );
   std -> add_action( "blessing_of_the_seasons" );
+  std -> add_action( "vanquishers_hammer", "if = buff.seraphim.up & runecarving.duty_bound_gavel | !runecarving.duty_bound_gavel | !talent.seraphim.enabled" );
```

Comment:
```suggestion
  std -> add_action( "vanquishers_hammer", "if=buff.seraphim.up&runecarving.duty_bound_gavel|!runecarving.duty_bound_gavel|!talent.seraphim.enabled" );
```
Not sure why I didn't notice the whitespace before.

Reply:
I actually don't know what would happen on profile regeneration if you left the whitespace in there. But I fear things would break, so better to remove them :-)